primitives/io/src/lib.rs

```diff
@@ -63,6 +63,25 @@ mod batch_verifier;
 #[cfg(feature = "std")]
 use batch_verifier::BatchVerifier;
 
+#[cfg(feature = "std")]
+#[macro_use]
+extern crate lazy_static;
+#[cfg(feature = "std")]
+use parking_lot::Mutex;
+#[cfg(feature = "std")]
+use sp_profiler::{Profiler, BasicProfiler, FileProfiler, TracingProfiler};
+#[cfg(feature = "std")]
+lazy_static! {
```

Comment:
A big no to this.

Reply:
I know this isn't the way to do it, but wanted a way to support profiling in situations where we don't have the client - eg. for isolated benchmarking of the runtime. Will figure out in due course

Reply:
The solution is to instantiate the tracer if you need it.